docs/src/content/docs/getting-started/usage.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Usage
+description: Basic usage of `@sheepdog/svelte`.
+---
+
+import { Tabs, TabItem, Code, Aside, LinkCard } from '@astrojs/starlight/components';
+
+The simplest way to use `@sheepdog/svelte` is the following
+
+<Tabs syncKey="examples">
+	<TabItem label="Dot notation">
+	```svelte
+	<script>
+		import { task, timeout } from '@sheepdog/svelte';
+
+    	const myTask = task(async () => {
+    		// async stuff
+    	});
+    </script>
+
+    <button on:click={()=>{
+    	myTask.perform();
+    }}>
+    	do async stuff
+    </button>
+    ```
+    </TabItem>
+    <TabItem label="Option notation">
+    ```svelte
+    <script>
+    	import { task, timeout } from '@sheepdog/svelte';
+
+    	const myTask = task(async () => {
+    		// async stuff
+    	}, { kind: "default" });
+    </script>
+
+    <button on:click={()=>{
+    	myTask.perform();
+    }}>
+    	do async stuff
+    </button>
+    ```
+    </TabItem>
+
+</Tabs>
+this will run your async function when the user press the button. *"But why not use a normal async function?"* i hear you ask.
+
+**Because the `myTask` variable that you used to call `perform` is also a svelte store that contains
+a lot of useful information's about your task!**
```

Comment:
```suggestion
**Because the `myTask` variable that you used to call `perform` is also a svelte store that contains
a lot of useful information about your task!**
```